### PR TITLE
Rearrange history models naming scheme

### PR DESCRIPTION
**Pull request by surabujin**
The goal is to make clear and consistent naming scheme.

### CODE REVIEW

File: src-java/base-topology/base-storm-topology/src/main/java/org/openkilda/wfm/share/mappers/HistoryMapper.java [hunk 1]
```diff
@@ -186,4 +180,38 @@ protected String mapPath(FlowPath path) {
...
+    public long mapTimestamp(Instant value) {
+        return value.getEpochSecond();
+    }
+
+    /**
+     * Map {@link Cookie} into {@link long}.
+     */
+    public long mapCookie(Cookie value) {
+        if (value == null) {
+            return 0;
```
Comment (sergii-iakovenko) [inline src-java/base-topology/base-storm-topology/src/main/java/org/openkilda/wfm/share/mappers/HistoryMapper.java:193]: Before, null cookie was mapped to null Long. What is the reason to implicitly introduce 0 as equivalent to null cookie? If null cookie is not acceptable then throw an exception.

File: src-java/base-topology/base-storm-topology/src/main/java/org/openkilda/wfm/share/mappers/HistoryMapper.java [hunk 2]
```diff
@@ -186,4 +180,38 @@ protected String mapPath(FlowPath path) {
...
+        }
+        return value.getValue();
+    }
+
+    /**
+     * Map {@link MeterId} into {@link long}.
+     */
+    public long mapMeterId(MeterId value) {
+        if (value == null) {
+            return 0;
```
Comment (sergii-iakovenko) [inline src-java/base-topology/base-storm-topology/src/main/java/org/openkilda/wfm/share/mappers/HistoryMapper.java:203]: Same as above with null cookie ~ 0.

File: src-java/kilda-model/src/main/java/org/openkilda/model/history/FlowStatusView.java [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+ */
+
+package org.openkilda.model.history;
+
+import lombok.Value;
+
+import java.time.Instant;
+
+@Value
+public class FlowStatusView {
```
Comment (sergii-iakovenko) [inline src-java/kilda-model/src/main/java/org/openkilda/model/history/FlowStatusView.java:23]: Please, add javadoc for the class.

**Review by sergii-iakovenko — COMMENTED** [review]
Looks good. Minor comments left.

File: src-java/base-topology/base-storm-topology/src/main/java/org/openkilda/wfm/share/mappers/HistoryMapper.java (hunk 1 above)
Comment (surabujin) [reply src-java/base-topology/base-storm-topology/src/main/java/org/openkilda/wfm/share/mappers/HistoryMapper.java:193]: Reasoin is the possible NPE - target objects use "long" not "Long" in corresponding fields.

**Review by dpoltavets — APPROVED** [review]

**Review by niksv — APPROVED** [review]

**Pull request merged**